_layouts/default.html

```diff
@@ -8,6 +8,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="apple-touch-icon" href="apple-touch-icon.png">
     <link rel="stylesheet" href="{{ site.baseurl }}/assets/css/main.css">
+    <link rel="shortcut icon" href="{{ "/favicon.ico" | prepend: site.baseurl | prepend: site.url }}" />
```

Comment:
Can this be changed to: `{{site.baseurl}}/favicon.ico` ?

Reply:
Agree with @ievavold on a more idiomatic way to express this. that said, hurray for that liquid filter cleverness, I smiled. :)